main.py

```diff
@@ -5,7 +5,8 @@ def subtract_two_nums(num1, num2):
 	return
 	
 def multiply_two_nums(num1, num2):
-	return
+	print(num1 * num2)
+	return num1 * num2
```

Comment:
you don't need the return line (not required), but it's fine!

LGTM - looks good to me